pengwin-setup.d/uninstall/x410.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+source $(dirname "$0")/uninstall-common.sh
+
+function main()
+{
```

Comment:
Put this in the same line as function. As a general rule, we use the Google Code style for bash scripts as a guideline